src/main/generic/consensus/primitive/Primitive.js

```diff
@@ -14,7 +14,7 @@ class Primitive {
     }
 
     toString() {
-        return this.toBase64();
+        return this.toHex().toUpperCase();
```

Comment:
I'm not sure about this change.
There are more classes inheriting from Primitive (such as Hashes) and we still want to rely on base64 there (e.g., since it is more efficient in terms of bytes per character). So, I would actually keep base64 here and just update the relevant lines where addresses are used.
Or another option could be to override the toString method for addresses.